posts/2024/2024_08_22_inigo_final_report.rst

```diff
@@ -0,0 +1,309 @@
+.. role:: custom-title
+
+.. |main-title| raw:: html
+
+   <span class="gsoc-title">Google Summer of Code Final Work Product</span>
+
+.. |project-abstract-title| raw:: html
+
+   <span class="gsoc-title">Project Abstract</span>
+
+.. |proposed-objectives-title| raw:: html
+
+   <span class="gsoc-title">Proposed Objectives</span>
+
+.. |modified-objectives-title| raw:: html
+
+   <span class="gsoc-title">Modified Objectives (Additional)</span>
+
+.. |objectives-completed-title| raw:: html
+
+   <span class="gsoc-title">Objectives Completed</span>
+
+.. |objectives-progress-title| raw:: html
+
+   <span class="gsoc-title">Objectives in Progress</span>
+
+.. |future-work-title| raw:: html
+
+   <span class="gsoc-title">Future Works</span>
+
+.. |open-source-tasks-title| raw:: html
+
+   <span class="gsoc-title">Open Source Tasks</span>
+
+.. |pull-requests-title| raw:: html
+
+   <span class="gsoc-title">Pull Requests</span>
+
+.. |conclusions-title| raw:: html
+
+   <span class="gsoc-title">Conclusions</span>
+
+.. |timeline-title| raw:: html
+
+   <span class="gsoc-title">Timeline</span>
+
```

Comment:
Sorry for the late review.

Add this here:

```
.. raw:: html

   <style>
   .custom-margin {
       margin-top: 20px;
       margin-bottom: 20px;
   }
   </style>

```

Reply:
Amazing, thanks